packages/admin/admin/src/SelectionRoute.tsx

```diff
@@ -9,10 +10,10 @@ interface IRouteParams {
 
 export function useSelectionRoute(): [ComponentType<IProps>, { id?: string; mode?: "edit" | "add" }, ISelectionApi] {
     const history = useHistory();
-    const parentMatch = useRouteMatch();
-    const match = useRouteMatch<IRouteParams>(`${parentMatch.path}/:id`);
+    const subRoutePrefix = useSubRoutePrefix();
```

Comment:
@nsams thanks for the fix. 

Do we want to implement it like this, or do we want to adjust the `useEditDialog()` that it supports a prefix - or do we want both?

Reply:
> Do we want to implement it like this

yes

> or do we want to adjust the useEditDialog() that it supports a prefix

Makes it easier to use, and it also possible to have two useEditDialog in the same component: yes, I'd say so

Reply:
> Makes it easier to use, and it also possible to have two useEditDialog in the same component: yes, I'd say so

then lets do this with a separat PR / Ticket --> https://vivid-planet.atlassian.net/browse/COM-2031